### DISCUSSION TITLE
Vamos compartilhar músicas/playlists?

### DISCUSSION THREAD

**Discussion by helderberto** [frontendbr/forum#1103] [Dicas]
Olá pessoal!

Vi que já existe uma issue sobre o assunto em #145, porém a proposta aqui é a troca para ampliarmos nosso conhecimento em bandas e estilos musicais. 💯 

Bom, vamos lá!

## Playlists/bandas (Spotify)

[Explosions In The Sky - Banda](https://open.spotify.com/artist/1uQWmt1OhuHGRKmZ2ZcL6p?si=7sWlY1LjQ_-akEoIMM2apA)
[Lofi Hip Hop Beats - Playlist](https://open.spotify.com/user/chillhopmusic/playlist/74sUjcvpGfdOvCHvgzNEDO?si=j_8MPNBHQIqCr78HQsZdSg)
[Ritalin (Hyper Focus) - Playlist](https://open.spotify.com/user/12177768210/playlist/3FAlO9LEpzUsNYPaUusfAz?si=3QbKZgiyQe6ueAdeIu1AlQ)
[DVLPR_FOCUS-v2.1 - Playlist](https://open.spotify.com/user/123472771/playlist/1KswcbMwcGC9RY84ZxNNi4?si=btCkadkYTEWFb_-MniIVcA)
[Programming - Playlist](https://open.spotify.com/user/mrbandler/playlist/3QK4ddLe1ADnI6lesTd27T?si=nzcC3zISQMeu4C0zSZ8VAQ)
[Rock Anthems - Playlist](https://open.spotify.com/user/spotify/playlist/37i9dQZF1DWXRqgorJj26U?si=cessAlPUQjW-yEof_RBdEg)
[Deep Focus - Playlist](https://open.spotify.com/user/spotify/playlist/37i9dQZF1DWZeKCadgRdKQ?si=e5MUbLhrQhqykceBvpoF2Q)

---

Essas são algumas das playlists que tenho acompanhado. :)

**Comment by tarciozemel** [comment]
- https://www.youtube.com/watch?v=LsBrT6vbQa8
- https://www.youtube.com/watch?v=twcOr043i4k
- https://www.youtube.com/watch?v=8gdhoj8HgaQ

**Reply by Hugobispo7** [reply]
Essas são as melhores do ano 21/22, link abaixo:
https://apptuts.bio/hugo-bispo-dos-s-133319
https://apptuts.bio/hugo-bispo-dos-s-133319
https://apptuts.bio/hugo-bispo-dos-s-133319
https://apptuts.bio/hugo-bispo-dos-s-133319

**Reply by marlonbzra** [reply]
https://www.youtube.com/channel/UC1fCBNrn3n3ZLrC15rCeMHQ

**Comment by vsppedro** [comment]
Será que cola um mix nesse post?

[Chuva](https://www.youtube.com/watch?v=nDq6TstdEi8) + [Moolingth Sonata](https://www.youtube.com/watch?v=4Tr0otuiQuU&lc=z22vgxfwcriyyr3ob04t1aokgth2r3vbxgybudnigobnrk0h00410)

Sugestão: abaixar o som da chuva até uns 10%.

**Reply by Hugobispo7** [reply]
Essas são as melhores do ano https://apptuts.bio/hugo-bispo-dos-s-133319
https://apptuts.bio/hugo-bispo-dos-s-133319
https://apptuts.bio/hugo-bispo-dos-s-133319

**Comment by helderberto** [comment]
@tarciozemel **lofi** é demais! Ouço quase que diariamente.

@VSPPedro curti demais o som da **Chuva**, valeu pela indicação!

**Reply by Hugobispo7** [reply]
https://apptuts.bio/hugo-bispo-dos-s-133319

**Comment by klarkc** [comment]
O estilo e ritmo depende do quanto quero me concentrar, de maneira geral a q eu mais escuto é

[Rick and Morty](https://open.spotify.com/user/1262817498/playlist/2Mkx1ytCuVVVtutFQomNKf?si=72UI_k3BR-qOSgteH4zxRQ)

Outras

[Mr. Robot Season 1](https://open.spotify.com/user/whoismrrobot/playlist/2Lv6c0lwFRTCA4npIxVJhX?si=OckqiMVRRR6IFg8Ng0rydA)
[Celtic Music](https://open.spotify.com/user/dfiechter2/playlist/5aSO2lT7sVPKut6F9L6IAc?si=FM1BzYsvQxaCAm9LTg3NoA)
[This is Gorillaz](https://open.spotify.com/user/spotify/playlist/37i9dQZF1DWSxQLa5KoDYW?si=8AnSkVc5Qs-iEnu8YqaggA)
[Mesmerizing Flute](https://open.spotify.com/user/sonyclassicalbenelux/playlist/2hhSMwyeYOTCwzKOFblQxW?si=2GyT22aFTK-38pMSla_c4A)

Quando o nível de concentração precisa ser máximo, eu escuto algum [álbum daqui](https://open.spotify.com/artist/1oCgMfQotTlv3jZxBIAwMl?si=28ON3lwPTLSjp1BGiBe4eg)

Bandas:
[Die Antwoord](https://open.spotify.com/artist/6urkHDoIVO1WO8vNIwcJmM?si=FvFh7hXrRxGLW6ttwkmSLQ)
[Avirl Lavigne](https://open.spotify.com/artist/0p4nmQO2msCgU4IF37Wi3j?si=o34RVO4nQheQuHxDa1ENnQ)
[Guns N' Roses](https://open.spotify.com/artist/3qm84nBOXUEQ2vnTfUTTFC?si=ugfKTDWURUipcwQdwsa14Q)
[Red Hot Chili Peppers](https://open.spotify.com/artist/0L8ExT028jH3ddEcZwqJJ5?si=A610BGtKR1mvs_WYxyIC0w)
[The Guess Who](https://open.spotify.com/artist/0cQuYRSzlItquYxsQKDvVc?si=1rd3TI3SSG-BQwHv0nZqtw)
[Pitty](https://open.spotify.com/artist/2dmQ0vMD3THLMcz7DsvfaT?si=E7N4ZWjASuuAnxSoe_iUOg)

**Reply by Hugobispo7** [reply]
https://apptuts.bio/hugo-bispo-dos-s-133319
https://apptuts.bio/hugo-bispo-dos-s-133319
https://apptuts.bio/hugo-bispo-dos-s-133319

**Comment by didiraja** [comment]
No geral, qualquer coisa instrumental serve. Post-rock, RnB, vale tudo. 

Mas, de maneira prática, eu fiz MUITOS exercícios da Alura, ouvindo essa [playlist de rock psicodélico](https://www.youtube.com/watch?v=iuWa5wh8lG0). E ultimamente, tenho curtido essa playlist aqui de [Vaporwave no Spotify](https://open.spotify.com/user/iam3/playlist/17HYiAIcwlDEg5RgVkm4L7?si=vz9QT2aZTRqZUC5Oh4a8lQ).

**Reply by Hugobispo7** [reply]
https://apptuts.bio/hugo-bispo-dos-s-133319

**Comment by leoamrl** [comment]
Pra quem gosta de eletrônica. Do low ao high bpm, segue abaixo alguns artistas que eu curto pra caramba.

## Techno
[Boris Brejcha](https://soundcloud.com/search?q=boris%20brejcha)

## Tech House
(gravadora) [Sublime Records](https://soundcloud.com/sublimemusicrecords)
[Dubdisko](https://soundcloud.com/search?q=dubdisko)
[FractaLL](https://soundcloud.com/search?q=fractall)
[Rocksted](https://soundcloud.com/search?q=rocksted)
[FlexB](https://soundcloud.com/search?q=flexb)

## House
Illusionise
Damien 'n Drix
Volac
Visage
Victor Lou
Breaking Beattz
Kvsh
Groove Delight
Gustavo Mota
Gabriel Boni
Shapeless
Paskal
Woak
Malaa

## Minimal
Devochka
Toporcov
Minimal Monkey
Henrique Camacho
BIG Smoke
Jon Mesquita
R3ckz3t

## Trance
Aura Vortex
Zanon
Major 7
Skazi
Blastoyz
Blazy
Gottinari
Belik Boom
Marambá

**Comment by felipefialho** [comment]
Curto as dicas do @willianjusten :D

**Comment by willianjusten** [comment]
Ixi...eu ouço taaanta coisa, que colocar uma lista aqui deixaria esse negócio enorme =o
Em cada post que faço no [meu blog](https://willianjusten.com.br/) desde quando comecei tem dica de uma música, então é só ir lá e pegar uma ahuahuahua

Nesse exato momento estou ouvindo [Vök - Before (Neelix Remix)](https://www.youtube.com/watch?v=3TCTJ_Rqyf4) é um remix de uma banda islandesa bem maneira, achei boa para concentrar no trabalho =p

**Reply by Hugobispo7** [reply]
https://apptuts.bio/hugo-bispo-dos-s-133319

**Comment by helderberto** [comment]
Conheci "Explosions in the sky" no blog do @willianjusten, banda foda demais!

**Reply by Hugobispo7** [reply]
https://apptuts.bio/hugo-bispo-dos-s-133319

**Comment by kelciocajueiro** [comment]
Conheci "Explosions in the Sky" agora e tô curtindo muito. @helderburato você sabe qual é o nome desse estilo musical? Conhece outras bandas do gênero?

**Reply by Hugobispo7** [reply]
https://apptuts.bio/hugo-bispo-dos-s-133319 as melhores 😜

**Comment by danilocandido** [comment]
Gosto de programar com essa de Rock Progressivo que é atualizada progressivamente. :goat:

[Rock Progressivo ](https://open.spotify.com/user/danilo07/playlist/6jjLut1DiQy4xFrL0JvqJX?si=YPeyd8YoQkesuro6OELi2w)

**Reply by Hugobispo7** [reply]
https://apptuts.bio/hugo-bispo-dos-s-133319 as melhores

**Comment by helderberto** [comment]
@kelciocajueiro dei uma googlada aqui, e a banda "Explosions In The Sky" é classificada como [Post rock](https://pt.wikipedia.org/wiki/Post-rock).

Bandas nesse estilo: [post-rock bands](https://en.wikipedia.org/wiki/List_of_post-rock_bands).

_Obs.: Não conheço muitas delas._

**Reply by Hugobispo7** [reply]
As melhires: https://apptuts.bio/hugo-bispo-dos-s-133319

**Comment by willianjusten** [comment]
Post Rock é um dos meus estilos de música favoritos. Aconselho muito uma banda chamada [Maybeshewill](https://www.youtube.com/watch?v=Gi1B8lZNHbI) se vocês curtem =)

**Comment by willianjusten** [comment]
Para quem curte Post-Rock, algumas bandas que eu aconselho:

- If These Trees Could Talk
- God is An Astronaut
- sleepmakeswaves
- 65daysofstatic
- Red Sparowes
- Collapse Under the Emprie
- Russian Circles
- This Will Destroy You
- Caspian
- pg.lost
- Mogwai (uma das primeiras bandas do estilo)

Provavelmente to esquecendo um monte, mas aí já são algumas xD

**Comment by marceloavf** [comment]
Algumas Post-Rock que curto, em adição as que o @willianjusten já citou:
- Pianos Become The Teeth
- Long Distance Calling
- Pelican
- Jakob
- Moonlit Sailor
- Joy Wants Eternety
- The Ocean
- Callisto

Post-Hardcore
- Touché Amoré
- La Dispute

Progressive Metal
- TesseracT
- Skyharbor
- Mechina
- Animals As Leaders
- Destiny Potato
- Pomegranate Tiger
- Protest The Hero

Progressive Rock
- Katatonia
- Karnivool

Hard Rock
- Smile Empty Soul
- Trapt
- Crossfade
- Staind
- Breaking Benjamin

New metal / Metalcore
- 36 Crazyfists
- Architects
- Breakdown of Sanity
- Northlane
- Veil Of Maya
- Novelists

Melodic Death Metal
- Be'lakor
- Insomnium
- Obscura

**Comment by ricardogouveia3** [comment]
A playlist da coworking onde eu trampo tá uma mistura de estilos, mas eu acho legal pra ouvir codando: 
https://open.spotify.com/user/ricardogouveia3/playlist/54MPVJCqzEdSL0BO9CaO0Q?si=xQKWOdkjT1KMweYbgRPbSQ

**Comment by RatoX** [comment]
Eu tenho umas playlists:

**RAP pra lembrar q nois é preto**
https://open.spotify.com/user/12167146129/playlist/5Tmg5d4Ww3qA5p3CZBDyhI

**Tudo que cê tem preconceito de ouvir mas se acaba de dançar quando toca!**
https://open.spotify.com/user/12167146129/playlist/5Z3LuXNssD7V09qyTPXZTg?si=OGZXbnu9Ql2OpaOnhyeeyQ

**Comment by felipefialho** [comment]
Quero sair do padrão e mandar [minha lista de modão no Spotify](https://open.spotify.com/user/12142158508/playlist/0IGbBFtZXPrWbEDGkOQfbr?si=fITyEJxAR86l8EBRejD2xQ), é minha trilha sonora toda sexta-feira e presença garantida no fim de festa com os amigos 😍

**Comment by helderberto** [comment]
Onde vai ser o churras @LFeh? hahah combinação perfeita 😄

**Comment by pedrocss** [comment]
Tenho acompanhado bastante música nacional:

- [Instituto](https://www.youtube.com/watch?v=Enh4PIKMpj4&t)
- [Curumin](https://www.youtube.com/watch?v=30qY4229CoY)
- [Tássia Reis](https://www.youtube.com/watch?v=K_DpNhv7gkI&list=PLOAgHAhw7Dikh9X74_kx02-kLs6pvmlbF)
- [BaianaSystem](https://www.youtube.com/watch?v=r9TgaoA3NhE&t)
- [Solveris](https://www.youtube.com/watch?v=f87mb9oqEEQ&t)
- [niLL](https://www.youtube.com/watch?v=JIPnf4ramgI&t)
- [DonL](https://www.youtube.com/watch?v=ET8FuO6ZgGk)
- [Criolo](https://www.youtube.com/watch?v=gGP2fQenAiY)
- [Kamau](https://www.youtube.com/watch?v=1e8wHX6bYBI)
- [Medulla](https://www.youtube.com/watch?v=4F0Qkf9JQ7I&list=PL9Y1H1m1NSettWdUu7-_UsqA0glnzmIdQ)
- [Selvagens à procura da lei](https://www.youtube.com/watch?v=8HHhq6AGNLE)
- [Vivendo do Ócio](https://www.youtube.com/watch?v=2IlLrjMR5S0)
- [Drik Barbosa](https://www.youtube.com/watch?v=w4RoaI1_qt4&list=PLPl5BbcRUuyR5YEJTJ815Vbybs9dLzTut)
- E muitos outros Baco Exu do Blues, Diomedes Chinasky, Emicida, Rael, Brisa Flow, Coruja BC1, Thiago Elniño, Síntese, LAY, Sant, Rashid, Kayuá, Sain, BK, Akira Presidente, Karol Konka, Well, Froid, Djonga, Eloy Polemico, MOB79, Inquérito, Maneva, Planta e Raiz, Natiruts, Cidade Verde Sound, KK Ousado, O Terno, Bivolt, Luccas Carlos, Rico Dalasam, Rodrigo Ogi, Solano Jacob, IZA, Yas Werneck, davzera, Rincon Sapiência, Amanajé Sound System, ÀTTØØXXÁ, Makalister, Marechal, Black Alien, Nego Max, e por aí vai.


Alguns sons internacionais:

- [Sam the kid](https://www.youtube.com/watch?v=TOpfu5F26T0&list=PL5F591AB3FD96E81F)
- [Stas](https://www.youtube.com/watch?v=QgngB58DE1I)
- [Tyler, The Creator](https://www.youtube.com/watch?v=N1w-hDiJ4dM&t=1s)
- [GURU](https://www.youtube.com/watch?v=8FzV21Lqd3A&list=PLoJR7HIEfo2i70gn_NWQ27F3Dk8iK0G_J)

**Comment by jhonnymoreira** [comment]
@marceloavf só banda linda, hein mano? <3

**Comment by jhonnymoreira** [comment]
Eu tenho uma forte preferência por ouvir Drum and Bass (liquid), pq tem vocais suaves e consigo abstrair a batida.

Comecei uma playlist no Spotify, tá com 2h de música: https://open.spotify.com/user/jhonnoso/playlist/2e9b9U4Zl6kna8nidoy6BG?si=xJRuZBpEQYewWE73KTqshA

Vou atualizar aos poucos, mas espero que gostem!

**Comment by raisiqueira** [comment]
Curto bastante a Ritalin Hiper Focus: https://open.spotify.com/user/12177768210/playlist/3FAlO9LEpzUsNYPaUusfAz?si=kqZLW7IoT4i6Xzz8Yh1ueA

**Comment by rapharma** [comment]
Antes queria compartilhar um infográfico interessante

[How Does Music Affect Work? [Infographic]](https://toggl.com/blog/how-does-music-affect-work-infographic)

### Playlist Instrumental

[Starbucks Jazz
](https://open.spotify.com/user/starbucks/playlist/2UPjWBRjJqMQY3LRV2aDTS?si=e9N3m_U2QXmlJCOm17N4Fg)

### Playlists Rock 

[This is Skillet](https://open.spotify.com/user/spotify/playlist/37i9dQZF1DZ06evO2qgZOg?si=CWFLa5GvQ8CR_rDUk8jTNA) 

[This is Oficina G3](https://open.spotify.com/user/spotify/playlist/37i9dQZF1DXdLXJfbLZljt?si=1dCcV_YkQJmqXf3AqvPVAQ)

**Comment by danielnunesdc** [comment]
Gosto dessa [Hora do Pesadelo](https://open.spotify.com/user/spotify/playlist/37i9dQZF1DX0H97kz3DdJ6?si=Dl4XMGB1TDaEtfhQOnIJHA)

**Comment by JacksonOsvaldo** [comment]
Dentre outras coisas, as mais "cosumidas":

https://open.spotify.com/user/jacksonosvaldo/playlist/0PLiNJdhhJVsQcEicW4GoV

https://open.spotify.com/user/jacksonosvaldo/playlist/6DLOByLTq7Gx63YirZ2DU2

https://open.spotify.com/user/jacksonosvaldo/playlist/5s3X3eyPskzZaStHkhcWPT

:)

**Comment by matheus-rosin** [comment]
Se quiser um rock limpão, agora estou ouvindo uma banda chamada Polyphia, um verdadeiro monstro. Outras duas brutais são Plini e Intervals.

Já se prefere um violão 0.12+, tem um estilo de execução ligeiramente desconhecido de nós, brasileiros: fingerstyle. Nomes: Erik Mongrain (pai de todos), Tommy Emmanuel, Mike Dawes, Andy McKee, Calum Graham, Pino Forastiere, Luca Stricagnoli, Petteri Sariola, Maneli Jamal e mais uma renca. Sample: https://www.youtube.com/watch?v=5dz1nF4Hdcs  https://www.youtube.com/watch?v=W_1LN_JFoDM

E se você é melancólico e curte sofrer, dá uma procurada por Hammock ;)

**Comment by rcbpgpg** [comment]
Fresquinha daqui de BH.

https://open.spotify.com/track/1RvbLntceW3cly66STNM7s?si=lnl_d8qRTzybhJ4KsOxOBQ

**Comment by rodrigets** [comment]
Pra quem  curte um Metal :metal: 

## Nowrong ##
https://open.spotify.com/album/7MqwARx3Es72uNS2ITj059

**Comment by praganavictor** [comment]
Estou fazendo essa de lofi: https://www.youtube.com/playlist?list=PL8gUHPip3lUmYUmU2aaRnWjXMrCOp66nk

Musica eletronica(boa pra jogar): https://www.youtube.com/playlist?list=PL8H85HKySx21UaDs0qoY5N-yTTU8R_4Qg

Musicas para fazer corno chorar(nao julguem kkk é mt boa): https://open.spotify.com/user/12151328780/playlist/0k1U1kHvbn2pGKhdfinn1P?si=YsG9JD0jTV2wrst2wBLeRg

**Comment by kvnol** [comment]
Meu negócio é black music, então: [Groove Nosso de Cada Dia](https://open.spotify.com/playlist/2Fa8VoowmqhT5XgAZR8zLv)

**Comment by gamarsh94** [comment]
Musica Boa ( SKY HIGH ) 

https://youtu.be/jPEPffsh1WI

**Comment by gabriel-hahn** [comment]
Minha [playlist](https://open.spotify.com/playlist/3yz5jIO6WBH5RTEStaSs8x?si=AExQJn5OQOKA6MMGveGZjw) com uma seleção geral de rock para vários gostos que fiz ao longo dos anos :metal:

**Comment by YasmimVieira** [comment]
Gosto de muitos estilos de música, mas quando estou programando escuto músicas mais calmas, como indie, músicas ambientes também e algumas que ajudam na concentração, como à abaixo:

https://www.youtube.com/watch?v=SiFNOQzRagk

uma vez estava estudando um pouco sobre como nosso cérebro funciona para concentração e ele possui ondas cerebrais que quando atingidas melhoram nossa concentração, essas ondas cerebrais podem ser atingidas através de  músicas como essa ou outras que faça você manter o foco. (li em um livro que tenho sobre anatomia, por isso n mando o link hehe)

**Comment by MichelPugliani** [comment]
Gosto de ouvir musicas, tranquilas pra estudar, editar, dormir e tals.
Esse canal aqui tem musicas top :+1: 
https://www.youtube.com/channel/UCo44OdnYjOjGhAXyY-j5sjA/featured
E estão postando musica todos os dias.

**Comment by MisterEdwardCoffe** [comment]
Fala ae pessoal . Curto bastante escutar um Lo-Fi / deep house . 
Deem uma olhada nesse canal .

Link Direto : https://www.youtube.com/channel/UC9UzS804Afw-U4ddVCye8jQ

Fã link : https://bit.ly/2Wl2ZK7

**Comment by AmandaFonseca** [comment]
Não sou organizada coloco tudo numa playlist só mas tem mais rock https://open.spotify.com/playlist/3Yl0IdmVYDPqQ3x0Z7iM8X?si=c9055ae029a5493c

**Comment by soficatXO** [comment]
Eu tenho muitas playlists ótimas no meu spotify 

Minding My Business: https://open.spotify.com/playlist/186ZM80niHEHBkWogNAFoo?si=6c98d48aac314d78
Brazilian Emo Scene: https://open.spotify.com/playlist/4zNRtbU9JuHnGMS9RfKMcA?si=11955be606634032
Tudo que ficou aqui: https://open.spotify.com/playlist/6Uigm181oFdTCXPONQ9Jcw?si=0fca531918b947cb
French vibes: https://open.spotify.com/playlist/5BqCymABP7FKLwFMfPgZFo?si=bdd004de3e1146b8

**Comment by MisterEdwardCoffe** [comment]
Ai Pessoal, aqui tem bastante música CALMA / AMBIENTE .... LO-FI  de primeira.

PLAYLIST DE LOFI PARA TRABALHAR/ CONCENTRAR/ PROGRAMAR
https://www.youtube.com/playlist?list=PLZbBXtH_wopXpufH55pB9Ow1BtMNDixxG

**Comment by vidadicas21** [comment]
Melhor Playlist de Pagode antigo

CLIQUE NO LINK ABAIXO

https://apptuts.bio/rafael-da-silva--128911

**Comment by nazarepiedady** [comment]
Ao desenvolvimento de software, juntamos a lógica, criatividade, imaginação e raciocínio, onde algumas delas nascem do puro processo intelectual ao qual chamamos de pensamentos e outras nascem da experiência adquirida pela contemplação do belo e compreensão da harmonia das coisas, e para isso nada melhor que musicas que contam-nos boas histórias e nos transportam para um estado positivo de perplexidade.

Com base nisso, eu recomendo ouvir música clássica, dos seguintes autores **Beethoven**, **Vivaldi**, **Mozart** e **Bach**.

Essas músicas podem ser achadas no canal [**HALIDONMUSIC**](https://www.youtube.com/channel/UCyOfqgtsQaM3S-VZnsYnHjQ)

**Comment by Adedrive** [comment]
**Essas são as melhores músicas e playlists  organizadas para curtir e encerrar o ano de 2021. 
Estou curtindo muito.** 

LISTAS NO LINK ABAIXO
https://apptuts.bio/admsom-130179

**Comment by nathanbalthazar** [comment]
Eu só consigo estudar se estiver escutando alguma playlist do canal [Lofi Girl](https://www.youtube.com/c/LofiGirl)

**Comment by felipe-heredia** [comment]
[Jazz & Blues](https://www.youtube.com/channel/UC9zktRjA2aI7cEZOfkDqmLg) esse canal se tornou essencial pra mim.

**Comment by Nick993** [comment]
Opa,

https://ouo.io/uEfmDVW - playlist

https://ouo.io/oxFGbw - playlist

https://ouo.io/46a8Kz - Cantor, L7

**Comment by Vini1011** [comment]
músicas que eu amo

https://7r6.com/yZL7
https://7r6.com/s0DJFFSN
https://7r6.com/5GWXib
https://7r6.com/cwPAIgA1

**Comment by ACCaneiro22** [comment]
oi
Compartilhando Playlists
https://apptuts.bio/ac-carneiro-141363

**Comment by felquis** [comment]
Alpha Portal, trance

https://music.youtube.com/watch?v=GjNJzf4jX44&list=OLAK5uy_kk5mQzMhRr_0yq03LjuMn15-4SZgvqZn0

**Comment by ricardo-ramos** [comment]
Uma das bandas novas de hard rock mais interessantes da atualidade... Power trio criado por irmãs e, em algumas músicas, a baterista também canta (quem toca bateria, principalmente, deve ter uma ideia do quanto isso é complicado)... e o resultado final é muito bom!

Referência sonora: hard rock moderno tipo Alter Bridge ou Shinedown, com um pouco de influência de Muse, e os vocais lembram Halestorm.

https://www.youtube.com/watch?v=Lh2eY0pe9X0

**Comment by lcs185** [comment]
http://usheethe.com/kF2N achei esse canal incrivel que faz musicas 8d!!!

**Comment by ghost** [comment]
Oi pessoal.

Algumas músicas/bandas/artistas/orquestras/grupos musicais/gêneros/estilos musicais que gosto/prefiro quando programo são essas:
- Soulfy - Son song
- Primus - The Devil Went Down to Georgia
- Portishead - Roads
- Red Hot Chili Peppers - Californication
- Descendents - I'm The One
- Björk - Army of Me
- Capital Inicial - A valsa do inferno
- Korn - Make Me Bad / In Between Days (feat. The Cure) 
- King Diamond - At the Graves
- Heart - Barracuda
- Ludwig van Beethoven - Für elise
- Toquinho - Aquarela
- Adoniran Barbosa - Trem das onze
- Raul Seixas - O dia em que a terra parou
- Cartola - O mundo é um moinho
- Smashing Pumpkins - Zero
- The cure - Friday I'm in Love
- Gorillaz - Clint Eastwood
- Evanescence - Even in Death
- The Gladiators - Soul Rebel
- Dir en grey - Afraid to die 
- Maximum the Hormone - What's up
- Malice Mizer - Ma Chérie
- Titãs - Tô cansado
- Biquini Cavadão - Tédio
- Cabine C - Lágrimas
- Black Sabbath - Snowblind
- Green Day - Boulevard of broken dreams
- Iron Maiden - Seventh Son of a Seventh Son
- Dio - Holy Diver 
- Alberto Nepomuceno - Abul(Dança Sacra do Véu)
- Heitor Villa-Lobo - Bachiana Brasileira No. 5 - II. Dansa (Martelo) - Allegretto
- Legião Urbana - Eduardo e Mônica
- Frank Sinatra - My way
- Dave Brubeck - Take Five
- Eric Clapton - Cocaine
- Nina Simone - Feeling good

*Espero ajudar de alguma forma aqui - iria acrescentar mais coisa, mas tentei focar apenas no que escuto com certa frequência para programar coisas*

**Comment by gabirlodis** [comment]
Eu conheçõ uma playlist perfeita cheia de músicas de rock e pop moderno nacional!! [https://open.spotify.com/playlist/0weKYGSvzMBiiJL89T4azR?si=0b124358e8 0c4858](https://open.spotify.com/playlist/0weKYGSvzMBiiJL89T4azR?si=0b124358e80c4858)

**Comment by Ninja1375** [comment]
> Olá pessoal!
> 
> Vi que já existe uma issue sobre o assunto em #145, porém a proposta aqui é a troca para ampliarmos nosso conhecimento em bandas e estilos musicais. 💯
> 
> Bom, vamos lá!
> 
> ## Playlists/bandas (Spotify)
> [Explosions In The Sky - Banda](https://open.spotify.com/artist/1uQWmt1OhuHGRKmZ2ZcL6p?si=7sWlY1LjQ_-akEoIMM2apA) [Lofi Hip Hop Beats - Playlist](https://open.spotify.com/user/chillhopmusic/playlist/74sUjcvpGfdOvCHvgzNEDO?si=j_8MPNBHQIqCr78HQsZdSg) [Ritalin (Hyper Focus) - Playlist](https://open.spotify.com/user/12177768210/playlist/3FAlO9LEpzUsNYPaUusfAz?si=3QbKZgiyQe6ueAdeIu1AlQ) [DVLPR_FOCUS-v2.1 - Playlist](https://open.spotify.com/user/123472771/playlist/1KswcbMwcGC9RY84ZxNNi4?si=btCkadkYTEWFb_-MniIVcA) [Programming - Playlist](https://open.spotify.com/user/mrbandler/playlist/3QK4ddLe1ADnI6lesTd27T?si=nzcC3zISQMeu4C0zSZ8VAQ) [Rock Anthems - Playlist](https://open.spotify.com/user/spotify/playlist/37i9dQZF1DWXRqgorJj26U?si=cessAlPUQjW-yEof_RBdEg) [Deep Focus - Playlist](https://open.spotify.com/user/spotify/playlist/37i9dQZF1DWZeKCadgRdKQ?si=e5MUbLhrQhqykceBvpoF2Q)
> 
> Essas são algumas das playlists que tenho acompanhado. :)

https://youtu.be/uscuveHJSkg?si=yCSU2OlF2N_6UxCs
